src/backends/fig_shapes.f90

```diff
@@ -4,6 +4,7 @@ module fig_shapes
     type, abstract :: shape
         type(RGB) :: fill_color
         type(RGB) :: stroke_color
```

Comment:
These values are not initialized @AnonMiraj, to do so, I recommend to set them to default values. For example for now, it could be a white or transparent background, and black fill color: 

```
type(RGB) :: fill_color = BLACK
type(RGB) :: stroke_color = WHITE
```